src/Tasks/Drush/Drush.php

```diff
@@ -38,28 +37,53 @@ class Drush extends Exec
      */
     public function siteInstall()
     {
+        switch ($this->databaseType) {
+            case 'sqlite3':
+            case 'sqlite':
+                $dbUrl = sprintf(
+                    "sqlite://sites/$this->sitesSubdir/files/.ht.sqlite"
```

Comment:
This is not the right way to construct the database URL. This is ignoring the database name and hardcoding it in some arbitrary location. Also we should have no need to switch on the database type. The database URL is adhering to a very specific scheme which is compatible with all database types.

I would instead use https://github.com/jakeasmith/http_build_url so we can have 1 line of code to generate valid URLs regardless of the database engine.